ParkingZoneApp/Controllers/ReservationController.cs

```diff
@@ -90,6 +92,10 @@ public IActionResult Reserve(ReserveVM reserveVM)
             {
                 var reservation = reserveVM.MapToModel();
                 reservation.UserId = User.FindFirstValue(ClaimTypes.NameIdentifier);
+
+                if (reservation.UserId is null)
```

Comment:
if inside the if little bit complex Kiss Principle is broken

Reply:
u suggest to remove it?

Reply:
noo find another way

Reply:
but it will be right if u remove it even because if user is not signed in controller doesnt work, and if user signed in of course there has to be user id it cant be null